libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -533,6 +538,14 @@ class YulUtilFunctions
 	/// signature: (array, newLen)
 	std::string resizeDynamicByteArrayFunction(ArrayType const& _type);
 
+	/// @returns the name of a function that cleans up elements of a storage byte array starting from pos
+	/// it will not copy elements in case of transformation to short byte array, and will not change array length
+	/// In case of pos is greater than len, doesn't do anything
+	/// In case of short byte array (< 32 bytes) doesn't do anything
+	/// Cleans up from first unused slot
+	/// signature: (array, len, pos)
```

Comment:
Having read the PR, I think it would be a bit easier to understand why it does things the way it does if it was `(array, oldLen, newLen)`. The function does not simply clear the area between `pos` and `len` - it does some have additional assumptions.

Reply:
I think `oldLen` and `newLen` can maybe mean that function does resizing. I'll rename `pos` to `startIndex`, same like with other function